addons/addon-environment-sc-ui/packages/environment-type-mgmt-ui/src/parts/environment-types/EnvTypeEditor.js

```diff
@@ -165,6 +165,7 @@ class EnvTypeEditor extends React.Component {
       case 'configurations':
         return (
           <ConfigStep
+            // data-testid="config-step"
```

Comment:
is this supposed to be commented out?

Reply:
No, it should be deleted. I didn't end up using it in the test but I wanted to test run the test to make sure it would work without this ID so I commented it out and forgot to delete it.